src/TradingBotConfig.py

```diff
@@ -8,7 +8,7 @@
 CONFIG_INPUT_MODE_IS_REAL_MARKET = True
 
 # Main ticker : Retrieves the next samples and processes them
-CONFIG_MAIN_TICK_DURATION_IN_MS = 200
+CONFIG_MAIN_TICK_DURATION_IN_MS = 10000

```

Comment:
The comment on line 14 suggests that this should be 10000 and not 200. What is the significance of it differing?